conans/test/functional/toolchains/apple/test_xcodedeps_components.py

```diff
@@ -105,10 +105,10 @@ def package_info(self):
         self.cpp_info.components["core"].includedirs.append("include")
         self.cpp_info.components["core"].libdirs.append("lib")
 
-        self.cpp_info.components["client"].libs = ["client"]
-        self.cpp_info.components["client"].includedirs.append("include")
-        self.cpp_info.components["client"].libdirs.append("lib")
-        self.cpp_info.components["client"].requires.extend(["core", "tcp::tcp"])
+        self.cpp_info.components["client-test"].libs = ["client"]
```

Comment:
my naming inspiration wasn't great at this point, so please feel free to suggest a better name for the component itself.